wait/http_test.go

```diff
@@ -562,7 +562,8 @@ func TestHttpStrategyFailsWhileGettingPortDueToNoExposedPorts(t *testing.T) {
 		},
 		StateImpl: func(_ context.Context) (*types.ContainerState, error) {
 			return &types.ContainerState{
-				Status: "running",
+				Status:  "running",
+				Running: true,
```

Comment:
Logic elsewhere actually looks for Running to be true and fails when it's simply Status == "running".  It might be worth going through and cleaning this up everywhere.